uiowa/uiowa
# Hotfix: Default content fixes

Updates
* Two image filenames that were preventing them from being properly installed.
* Resolves #2330

# To test
* `composer install`
* `blt drupal:install --site=default`
* Check that home is nid = 1
* Build passed without image errors: https://travis-ci.com/github/uiowa/uiowa/builds/189778711
